doc/source/ray-core/ray-dashboard.rst

```diff
@@ -6,74 +6,280 @@ Ray provides a web-based dashboard for monitoring and debugging Ray applications
+
+SANG-TODO Image of the profilng buttons with a red circle.
+
+.. _dash-workflow-job-progress:
+
+Advanced Task/Actor Breakdown
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The job page allows you to see tasks and actors broken down by their states. 
+Tasks and actors are grouped and nested by default. You can see the nested entries by clicking the expand button.
+
+Tasks/actors are grouped and nested by the following criteria.
+
+- All tasks and actors are grouped together, and you can view individual entries by expanding the corresponding row.
+- Tasks are grouped by their ``name`` attribute (e.g., ``task.options(name="<name_here>").remote()``).
+- Child tasks (nested tasks) are nested under their parent task's row.
+- Actors are grouped by their class name.
+- Child actors (actors created within an actor) are nested under their parent actor's row.
+- Actor tasks (remote methods within an actor) are nested under the actor for the corresponding actor method.
+
+TODO-SANG Add images
+
+.. note:: 
+
+  Ray dashboard can only display/retrieves up to 10K tasks at a time. If there are more than 10K tasks from your job,
+  they are unaccounted. The number of unaccounted tasks are available from the task breakdown.
+
+Task Timeline
+~~~~~~~~~~~~~
+
+The :ref:`timeline API <ray-core-timeline>` is available from the dashboard. 
+You can download the chrome tracing file by clicking the download button. You can either go to ``chrome://tracing`` or the `Perfetto UI <https://ui.perfetto.dev/>`_ and drop the downloaded chrome tracing file to see the timeline visualization of Ray tasks and actors.
+
+TODO-SANG Add images
+
+Ray Status
+~~~~~~~~~~
+
+The job page displays the output of the helpful CLI tool ray status, which shows the autoscaler status of the Ray cluster.
+
+The left page shows the autoscaling status, including pending, active, and failed nodes. 
+The right page displays the cluster's demands, which lists demanded resources that cannot be scheduled to the cluster at the moment. This page is useful for debugging resource deadlocks or slow scheduling.
+
+TODO-SANG Add images
+
+.. note:: 
+
+  The output shows the aggregated information across the cluster. If you run more than on job, the demands could be wrong.
+
+.. _dash-workflow-state-apis:  
+
+Task Table, Actor Table, Placement Group Table
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The dashboard shows a table with the status of the job's tasks, actors, and placement groups. 
+This information is the same as what you would get from the :ref:`Ray state APIs <state-api-overview-ref>`.
+
+.. _dash-node-view:
+
+Cluster View
+------------
+
+The cluster view visualizes hierarchical relationship of
+machines (nodes) and workers (processes). Each host consists of many workers, and
+you can see them by clicking the + button. This also shows the assignment of GPU resources to specific actors or tasks.
```

Comment:
Do we currently show "assignment of GPU resources to specific actors or tasks."?

Reply:
It is supported now